### PR TITLE
fix: compile

### PR DESCRIPTION
**Pull request by kir19890817**

### CODE REVIEW

**Comment by developer0hye** [comment]
@kir19890817 
Thanks for your contribution!

Can you explain your PR?

File: YoloLabel.pro [hunk 1]
```diff
@@ -20,7 +20,7 @@ DEFINES += QT_DEPRECATED_WARNINGS
 # You can also select to disable deprecated APIs only up to a certain version of Qt.
 #DEFINES += QT_DISABLE_DEPRECATED_BEFORE=0x060000    # disables all the APIs deprecated before Qt 6.0.0
 
-CONFIG += c++11
```
Comment (pktiuk) [inline YoloLabel.pro:23]: @developer0hye 

Without this line I get:

```
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtWidgets -isystem /usr/include/x86_64-linux-gnu/qt5/QtGui -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -I. -I. -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++ -o mainwindow.o mainwindow.cpp
mainwindow.cpp: In member function 'void MainWindow::on_tableWidget_label_cellClicked(int, int)':
mainwindow.cpp:446:64: warning: unused parameter 'column' [-Wunused-parameter]
  446 | void MainWindow::on_tableWidget_label_cellClicked(int row, int column)
      |                                                            ~~~~^~~~~~
g++ -c -pipe -O2 -std=gnu++11 -Wall -W -D_REENTRANT -fPIC -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtWidgets -isystem /usr/include/x86_64-linux-gnu/qt5/QtGui -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -I. -I. -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++ -o label_img.o label_img.cpp
label_img.cpp: In member function 'virtual void label_img::mouseReleaseEvent(QMouseEvent*)':
label_img.cpp:74:48: warning: unused parameter 'ev' [-Wunused-parameter]
   74 | void label_img::mouseReleaseEvent(QMouseEvent *ev)
      |                                   ~~~~~~~~~~~~~^~
label_img.cpp: In member function 'void label_img::setContrastGamma(float)':
label_img.cpp:377:18: error: 'clamp' is not a member of 'std'
  377 |         s = std::clamp(s, 0, 255);
      |                  ^~~~~
make: *** [Makefile:453: label_img.o] Error 1
```

**Comment by developer0hye** [comment]
@kir19890817 @pktiuk LGTM
Thanks!

**Comment by kir19890817** [comment]
@developer0hye thanks for explanation, didn't notice the thread!

**Pull request merged**